scheduling.md

```diff
@@ -278,6 +278,15 @@ To indicate that the task should run on only one server, use the `onOneServer` m
                     ->at('17:00')
                     ->onOneServer();
 
+Sometimes you will need to schedule the dispatch of same Job Class multiple times, but using different parameters. If you are using `onOneServer` option, only the first call will be dispatched. To avoid this behaviour, you need to define a unique name in each task:
```

Comment:
```suggestion
Sometimes you will need to schedule the dispatch of the same Job Class multiple times, but using different parameters. If you are using `onOneServer` option, only the first call will be dispatched. To avoid this behaviour, you need to define a unique name for each task:
```

Reply:
Thanks @dammy001 for reviewing